Localizations/en_US.json

```diff
@@ -173,39 +173,39 @@
     "shader_asset_format_disassembly": "Disassembly",
     "shader_asset_format_disassembly_description": "Export the shader as disassembly. Experimental and prone to breakage. This does not compile in the editor.",
     "shader_asset_format_dummy": "Dummy Shader",
-    "shader_asset_format_dummy_description": "Export the shader as a dummy shader. Although it preserves data like the Properties and FallBack, it uses general, opaque shader code.",
-    "shader_asset_format_yaml": "Yaml Asset",
-    "shader_asset_format_yaml_description": "Export the shader as a yaml asset. Experimental and only usable for viewing in the editor. The editor can randomly corrupt these files.",
+    "shader_asset_format_dummy_description": "Export the shader as a dummy shader. Although it preserves data like the Properties and Fallback, it uses general, opaque shader code.",
```

Comment:
My bad.

Reply:
Let's make properties and fallback lowercase.

Reply:
Sure!